glide.yaml

```diff
@@ -152,3 +152,5 @@ import:
   - pkg/api/v1
 - package: github.com/ajg/form
   version: ^1.5.0
+- package: github.com/getsentry/raven-go
```

Comment:
It's not enough. Add github.com/certifi/gocertifi too.

Reply:
tagged it as well